src/Components/RequestList.tsx

```diff
@@ -111,6 +111,18 @@ export const RequestList = (props: RequestListProps) => {
                             onClick={() => setSelectionMode(!selectionMode)}>
                             {selectionMode ? <Text id="cancel" /> : <Text id="selection-mode" />}
                         </button>
+
+                        {!selectionMode && (
+                            <>
+                                <button
```

Comment:
Since you just added the button to the `flex` container, I think with 3 buttons now there needs to be a small `column-gap` in between the buttons for small screens. Please add that.

Reply:
To have a consistent `column-gap` on all page i would like to use that number of pixels that are used "everywhere" but i discovered that you use 5px, 8px, 10px and i could not figure out your pattern. I go with `5px` as other elements on that page use 5px.